core/lib/core_web/ui/navigation/tabbar.ex

```diff
@@ -2,13 +2,14 @@ defmodule CoreWeb.UI.Navigation.Tabbar do
   @moduledoc false
   use CoreWeb.UI.Component
 
+  prop(initial_tab, :integer, required: true)
   prop(size, :atom, default: :wide)
 
   alias CoreWeb.UI.Navigation.{TabbarWide, TabbarNarrow}
 
   def render(assigns) do
     ~H"""
-    <div>
+    <div id="tabbar" initial-tab={{@initial_tab}} phx-hook="Tabbar">
```

Comment:
Maybe use a [data attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)? This avoid collisions with updated specs & has a slightly nicer API.

Reply:
Changed in my next PR